packages/rx_bloc_cli/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## [2.5.0]
```

Comment:
2.4 is not released so you can add your feature to it

Reply:
I have move the comment to the 2.4.0 version